### PR TITLE
New test for CASSANDRA-11381: Node running with join_ring=false and authentication can not serve requests

### PR DESCRIPTION
**Pull request by michaelsembwever**
Follow up to CASSANDRA-11381 and https://github.com/riptano/cassandra-dtest/pull/1479

I think the concerns raised previously [here|https://github.com/riptano/cassandra-dtest/pull/1479#issuecomment-315443711] have been addressed. fyi @jkni , @ptnapoleon

### CODE REVIEW

File: auth_join_ring_false_test.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+
+    def test_login_existing_node(self):
+        """
+        * Launch a three node cluster
+        * Restart the third node in `join_ring=false` mode
+        * Connect as the default user/password
+        * Verify that default user w/ bad password gives AuthenticationFailed exception
+        * Verify that bad user gives AuthenticationFailed exception
+        #"""
```
Comment (ptnapoleon) [inline auth_join_ring_false_test.py]: Seems like there's a loose # here?

Comment (michaelsembwever) [reply auth_join_ring_false_test.py]: oops. fixed.

**Comment by michaelsembwever** [comment]
@ptnapoleon ping

**Comment by michaelsembwever** [comment]
>  I believe pep8 will complain about the lack of a new line at the bottom

I did not know about that! Fixed.

Will merge.

**Pull request merged**